src/runtime/runtime.js

```diff
@@ -424,6 +430,14 @@ export class ConfiguredRuntime {
     });
   }
 
+  /** override */
+  showAbbrvOffer() {
+     return this.documentParsed_.then(() => {
+       const flow = new AbbrvOfferFlow(this);
```

Comment:
Need to import AbbrvOffersFlow.
Can you test this in the UI?

You might want to expose this in the runtime under createPublicRuntime(), so can test in the UI.